Tasks/AzureCLIV2/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vsts-tasks-azurecli",
-  "version": "1.0.0",
```

Comment:
this version is not needed to be changed. You should change version in task.json